firmware/config/boards/common_make.sh

```diff
@@ -17,7 +13,7 @@ mkdir -p .dep
 make -j$(nproc) -r PROJECT_BOARD=$PROJECT_BOARD PROJECT_CPU=$PROJECT_CPU
 [ -e build/rusefi.hex ] || { echo "FAILED to compile by $SCRIPT_NAME with $PROJECT_BOARD $DEBUG_LEVEL_OPT and $EXTRA_PARAMS"; exit 1; }
 if [ "$USE_OPENBLT" = "yes" ]; then
-  make openblt
+  make openblt PROJECT_BOARD=$PROJECT_BOARD PROJECT_CPU=$PROJECT_CPU
```

Comment:
@dron0gus thoughts? I'd rather be explicit about what we pass to openblt makefile rather than implicit with exported vars.